apps/tasks/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Could you replace this file with a project.json instead? I don't know why, but NX is not recognizing the targets here

Reply:
See the project.json on the sugarfoot

Reply:
done at [4c324d5](https://github.com/tassioFront/fubs/pull/21/commits/4c324d52d4bcfd795800a8a2c40e186061e8dd0f)